diff --git a/test/dtransform.c.orig b/test/dtransform.c
index 76db260..34ecd4f 100644
--- a/test/dtransform.c.orig
+++ b/test/dtransform.c
@@ -122,9 +122,8 @@ const int transformData[ROWS][COLS] = {{36, 31, 25, 19, 13, 7, 1, 5, 11, 16, 22,
     {                                                                                                        \
         TYPE        array[ROWS][COLS];                                                                       \
         const char *f_to_c = "(5/9.0)*(x-32)";                                                               \
-        /* utrans is a transform for unsigned types: no negative numbers involved and results are < 255 to   \
-         * fit into uchar */                                                                                 \
-        const char *utrans = "((x+100)/4)*3";                                                                \
+        /* utrans is a transform for char types: numbers are restricted from -128 to 127, fits into char */  \
+        const char *utrans = "(x/4+25)*3";                                                                   \
                                                                                                              \
         hid_t       dataspace, dxpl_id_f_to_c, dxpl_id_utrans, dset, dset_nn, dt_nn;                         \
         H5T_order_t order;                                                                                   \
