diff --git a/test/Makefile.in b/test/Makefile.in
index c2a5ee1f42..734da89e0a 100644
--- a/test/Makefile.in
+++ b/test/Makefile.in
@@ -1590,8 +1590,8 @@ check_SCRIPTS = $(TEST_SCRIPT)
 # As an exception, long-running tests should occur earlier in the list.
 # This gives them more time to run when tests are executing in parallel.
 TEST_PROG = testhdf5 \
-           cache cache_api cache_image cache_tagging lheap ohdr \
-           stab gheap evict_on_close farray earray btree2 fheap \
+           cache_api cache_tagging lheap ohdr \
+           stab gheap evict_on_close farray earray btree2 \
            pool accum hyperslab istore bittests dt_arith page_buffer \
            dtypes dsets chunk_info cmpd_dset cmpd_dtransform filter_fail extend direct_chunk \
            external efc objcopy objcopy_ref links unlink twriteorder big mtime \
@@ -1599,7 +1599,7 @@ TEST_PROG = testhdf5 \
            flush1 flush2 app_ref enum set_extent ttsafe enc_dec_plist \
            enc_dec_plist_cross_platform getname vfd ros3 s3comms hdfs ntypes \
            dangle dtransform reserved cross_read freespace mf vds file_image \
-           unregister cache_logging cork swmr thread_id vol timer
+           unregister cache_logging cork thread_id vol timer
 
 
 # These programs generate test files for the tests.  They don't need to be
